mint-choc-chip-skyblade/sshd-rando
# Hold to skip cutscenes

## What does this PR do?
Allows you to hold the minus button to skip cutscenes. Also makes the skip prompt in the bottom left corner always visible during the cutscene (required to allow the holding to work but also makes more sense for rando)

## How do you test this changes?
I tested entering the Demise fight and the skip prompt was immediately visible. I tested holding minus into the load and the cutscene immediately skipped and I was able to begin the fight. I also tested to make sure other non-skippable events work as expected (opening doors, sleeping, etc...)
